lang/en/docs/tutorials/dft/chemical/molecule-surface.md

```diff
@@ -0,0 +1,89 @@
+
+Since in this example the plane of the 2D benzene molecule and the gold (211) surface are already parallel to each other, a simple [translation](../../../materials-designer/3d-editor/editor-actions/move-rotate-atoms.md#translation) of the benzene atoms on top of the surface should suffice.
+
+## Exit Multi-Materials 3D Editor
+
+After the correct desired positioning of the benzene molecule on top of the gold surface, the user should now [exit](../../../materials-designer/3d-editor/edit.md#exit-the-editor) the Multi-Materials 3D Editor, and return to the original Materials Designer interface.
+
+The user will notice that a new material entry, called "New Material" by default, has now been created automatically and is listed within the [left-had items list sidebar](../../../materials-designer/sidebar-items.md) of the Materials Designer interface. It contains the combined benzene-gold surface crystallographic structure, as a new single material entity.
+
+This new entry should first be [renamed](../../../materials-designer/sidebar-items.md#edit-name-of-item) to a more memorable form, and should finally be [saved](../../../materials-designer/header-menu/input-output/save.md) via the ["Input/Output" Menu](../../../materials-designer/header-menu/input-output.md) located at the top-left corner into the account-owned materials [collection](../../../accounts/collections.md), as a new material structure entry which is distinct from both the original isolated benzene molecule and gold structure.
+
+## Run Further Reaction Profile Calculations
+
+The user is now free to run reaction energy profile calculations on the newly generated benzene-gold surface system, in order to calculate the size of the corresponding activation energy barrier for a chemical adsorption process of the molecule into the surface for instance. 
+
+The [Nudged Elastic Band (NEB)](../../../models/auxiliary-concepts/nudged-elastic-band.md) method is particularly recommended in the current context of reaction energy profile chemistry calculations. We offer two alternative approaches for implementing the NEB method on our platform, based on the use of the [VASP](../../../software-directory/modeling/vasp/overview.md) or [Quantum ESPRESSO](../../../software-directory/modeling/quantum-espresso/overview.md) modeling engines, which are narrated in two separate tutorials accessible [here](reaction-profile-vasp.md) and [here](reaction-profile-qe.md) respectively.
```

Comment:
"is particularly recommended in the current context of" => "can be used for"
"chemistry calculations" => "calculations"